proxysql-admin

```diff
@@ -3013,6 +3058,7 @@ function parse_args() {
   readonly SYNCMULTICLUSTERUSERS
   readonly QUICK_DEMO
   readonly UPDATE_CLUSTER
+  readonly UPDATE_MYSQL_VERSION
```

Comment:
Comment doesn't belong here, but nowhere else to put this.  Where is the initial declaration of UPDATE_MYSQL_VERSION, plus setting it to the default value?

Reply:
Declared UPDATE_MYSQL_VERSION variable.